pkgs/development/python-modules/py7zr/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  brotli,
+  buildPythonPackage,
+  fetchFromGitHub,
+  inflate64,
+  lib,
+  multivolumefile,
+  psutil,
+  pybcj,
+  pycryptodomex,
+  pyppmd,
+  pyzstd,
+  setuptools,
+  setuptools-scm,
+  texttable,
+}:
+
+buildPythonPackage rec {
```

Comment:
I think packages that are both libraries and applications should get a top-level package with the same name that uses `toPythonApplication`.